### PR TITLE
Makefile fix

### PR DESCRIPTION
**Pull request by mchindri**
This changes were tested with iio_demo and daq2 to build both in terminal and in sdk

### CODE REVIEW

**Review by amiclaus — COMMENTED** [review]
Nitpick:
for `tools:scripts: Fix build errors` the commit message should comply the 50/72 format rule.

File: tools/scripts/linux.mk [hunk 1]
```diff
@@ -109,8 +109,6 @@ LIB_TINYIIOD_PATH = $(LIBRARIES)/libtinyiiod/build
 LIB_TINYIIOD = tinyiiod
 TINYIIOD_STD_TYPES = _USE_STD_INT_TYPES
 CFLAGS += -D $(TINYIIOD_STD_TYPES)
-EXTRA_LIBS += $(LIB_TINYIIOD)
-EXTRA_LIBS_PATHS += $(LIB_TINYIIOD_PATH)
```
Comment (scuciurean) [inline tools/scripts/linux.mk:113]: If those are removed, what's happening with 
 `
ADD_LIBRARIES = $(foreach lib, $(EXTRA_LIBS), $(call add_library_cmd,$(lib)))
ADD_LIBRARIES_PATH = $(foreach path, $(EXTRA_LIBS_PATHS), $(call lib_serach_path_cmd,$(path)))
`
At the moment, the projects don't use the generic.mk file

Comment (mchindri) [reply tools/scripts/linux.mk:113]: They are added by create_project.tcl . That's the error was to add the here too, causing the duplication error.

**Comment by mchindri** [comment]
v2:
 - Fix commit message to fit 50/72 rule

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

**Pull request merged**